R/getResampleExtract.R

```diff
@@ -0,0 +1,35 @@
+getResampleExtract2 = function(learner) {
+  UseMethod("getResampleExtract2")
+}
+
+getResampleExtract2.NULL = function(learner) {
+  stopf("Wrapper without underlying Learner.")
+}
+
+getResampleExtract2.Learner = function(learner) {
+  NULL
+}
+
+getResampleExtract2.BaseWrapper = function(learner) {
+  getResampleExtract2(learner$next.learner)
+}
+
+getResampleExtract2.TuneWrapper = function(learner) {
+  c(list(TuneResult = getTuneResult), getResampleExtract2(learner$next.learner))
+}
+
+getResampleExtract2.FeatSelWrapper = function(learner) {
+  c(list(FeatSelResult = getFeatSelResult), getResampleExtract2(learner$next.learner))
+}
+
+getResampleExtract2.FilterWrapper = function(learner) {
+  c(list(FilteredFeatures = getFilteredFeatures), getResampleExtract2(learner$next.learner))
+}
+
+getResampleExtract = function(learner){
```

Comment:
We find out what we can extract by default for each wrapper. For multiple wrapped learners we can extract more than just one information so we collect all usable functions and then build one function which returns a list. Compare with [this tutorial part](http://mlr-org.github.io/mlr-tutorial/devel/html/nested_resampling/index.html#filter-methods-with-tuning).